pennylane/qchem/convert.py

```diff
@@ -1050,21 +1100,11 @@ def _dmrg_state(wavefunction, tol=1e-15):
 
     **Example**
 
-    >>> from pyscf import gto, scf, mcscf
-    >>> from pyblock2.driver.core import DMRGDriver, SymmetryTypes
-    >>> from pyblock2._pyscf.ao2mo import integrals as itg
-    >>> mol = gto.M(atom=[['H', (0, 0, 0)], ['H', (0,0,0.71)]], basis='sto3g')
-    >>> mf = scf.RHF(mol).run()
-    >>> mc = mcscf.CASCI(mf, 2, 2)
-    >>> ncas, n_elec, spin, ecore, h1e, g2e, orb_sym = itg.get_rhf_integrals(mf, mc.ncore, mc.ncas, g2e_symm=8)
-    >>> driver = DMRGDriver(scratch="./tmp", symm_type=SymmetryTypes.SU2)
-    >>> driver.initialize_system(n_sites=ncas, n_elec=n_elec, spin=spin, orb_sym=orb_sym)
-    >>> mpo = driver.get_qc_mpo(h1e=h1e, g2e=g2e, ecore=ecore)
-    >>> ket = driver.get_random_mps(tag="GS")
-    >>> wavefunction = driver.get_csf_coefficients(ket)
+    >>> import numpy as np
+    >>> wavefunction = ( [[0, 3], [3, 0]], np.array([-0.10660077,  0.9943019 ]))
     >>> wf_dmrg = _dmrg_state(wavefunction, tol=1e-1)
     >>> print(wf_dmrg)
```

Comment:
```suggestion
    >>> wf_dmrg
```

Reply:
Doesn't print for me without the print statement